src/content/structured/components/badge/code.mdx

```diff
@@ -305,124 +305,195 @@ export const snippetsMaxNumber = [
       slot="badge"
       variant="info"
     />
-    Default
+    Coffee Orders
   </IcButton>
 </div>`,
   },
 ];
 
 <ComponentPreview snippets={snippetsMaxNumber}>
-  <div style={{padding:"16px"}}>
-    <IcButton variant="secondary" style={{marginRight:"16px"}}>
-      <IcBadge
-        text-label="1000"
-        max-number="99"
-        slot="badge"
-        variant="info"
-       />
-      Default
+  <div style={{ padding: "16px" }}>
+    <IcButton variant="secondary" style={{ marginRight: "16px" }}>
+      <IcBadge text-label="1000" max-number="99" slot="badge" variant="info" />
+      Coffee Orders
     </IcButton>
   </div>
 </ComponentPreview>
 
-## Position near
+### Position near
```

Comment:
This section should be updated to show different badge positioning options, not different sized badges. 

Re-title section to 'Positioning'. 

In component demo show two chips, one with position near and one with position far.